package.json

```diff
@@ -142,7 +147,7 @@
       "debug/variables/context": [
         {
           "command": "memory-inspector.show-variable",
-          "when": "canViewMemory && memory-inspector.canRead"
+          "when": "memory-inspector.canRead"
```

Comment:
I think I'd prefer to keep the current behavior of this command intact:
 - The changes make this function polyvalent: rather than consistently showing where the variable resides, it may interpret the variable's value.
 - It may cause errors / failures: an adapter providing a memory reference is secure evidence that we can work with that variable; the other use cases involve more guess work.

On the other hand, I think having the option to follow a pointer visible in the variables view would be a nice complement to the new functionality in the view. Is there any way that we can use context keys or other mechanisms to try to control the availability of a command specifically for that?

One idea that occurs to me is that we could use the [`in` operator](https://code.visualstudio.com/api/references/when-clause-contexts#in-and-not-in-conditional-operators) and use our current snooping on variables request traffic to identify pointers and add them to a context key we control.

Reply:
I've removed this feature from the second draft.

To my knowledge, we can't set context keys for individual items in the debug variables view.